03-Data_Security.Rmd

```diff
@@ -228,47 +226,65 @@ These algorithms involve mathematical operations to encrypt the data. See this v
+ottrpal::include_slide("https://docs.google.com/presentation/d/1SRokLaGAc2hiwJSN26FHE0ZEEhPr3KQdyMICic8kAcs/edit#slide=id.g2134f5bcd65_0_0")
+```
 
-3) The browser checks the certificate to make sure that it comes from a trusted certificate authority and that the certificate has not expired or been revoked. The browser also checks that the names on the certificate  and the website match.  It then sends the server an encrypted session key based on the public key.
+#### SSL Socket Layer
 
-4) The server then decrypts the session key using the private key and sends an encrypted acknowledgment to start an encrypted session.
+This is what the s in http*s* is for. This indicates that data from the server is using a form of  encryption and that the website server has a special SSL certificate from a trusted certificate authority that verified that the server can be trusted. There is actually a lot happening behind the scenes!
 
-5) The server and browser can now transmit encrypted data back and forth.
+According to @digicert this involves the following steps:
 
+1. A web browser (like chrome) connects to a web server (website) that is secured with SSL (it starts with https instead of http). The browser asks the server to identify itself.
+2. The server sends a copy of the SSL certificate and the public key.
+3. The browser checks the certificate to make sure that it comes from a trusted certificate authority and that the certificate has not expired or been revoked. The browser also checks that the names on the certificate  and the website match.  It then sends the server an encrypted session key based on the public key.
+4. The server then decrypts the session key using the private key and sends an encrypted acknowledgment to start an encrypted session.
+5. The server and browser can now transmit encrypted data back and forth.
 
 Note that newer versions of this process are called Transport Layer Security (TLS).
 
 #### SSH Secure Shell
 
-SSH is also a process for securely transmitting files from one computer to another using encryption. However, in this case it can be done using  a command line interface instead of a browser. SSH also does not involve certificate. 
+SSH (secure shell) is also a process for securely transmitting files from one computer to another using encryption. However, in this case it can be done using  a command line interface instead of a browser. SSH also does not involve certificate.
```

Comment:
```suggestion
SSH (secure shell) is also a process for securely transmitting files from one computer to another using encryption. However, in this case it can be done using a command line interface instead of a browser. SSH also does not involve certificates.
```

Reply:
This all looks great @cansavvy! :)